game-core/src/main/java/games/strategy/triplea/delegate/RandomStartDelegate.java

```diff
@@ -94,7 +94,7 @@ private void setupBoard() {
       if (currentPickingPlayer == null || !playersCanPick.contains(currentPickingPlayer)) {
         currentPickingPlayer = playersCanPick.get(0);
       }
-      if (!Interruptibles.sleep(250)) {
```

Comment:
I actually don't know if its necessary which is why I didn't just remove it entirely but left a small sleep in case their is some reason from a painting the UI tiles that needs it. Probably better as a follow up if someone wants to test just removing it entirely and testing a bunch of maps.

Reply:
If you're feeling up for a it, a TODO comment with that question/context could be a good placeholder .

Reply:
Good idea. Added.